plugins/frontend/backstage-plugin-prometheus/src/hooks/usePrometheus.ts

```diff
@@ -132,10 +132,23 @@ export function useAlerts(alerts: string[] | 'all') {
     }, []);
   if (value && value.status !== 'error') {
     const rules = value.data.groups.flatMap(it => it.rules);
+    const labels = getLabels(entity);
+    const parsedLabels = labels
+      ? Object.fromEntries(labels.split(',').map(label => label.split('=')))
+      : undefined;
```

Comment:
Wouldn't this be better as: 
```suggestion
    const parsedLabels = labels && Object.fromEntries(labels.split(',').map(label => label.split('=')))
```

Please also make sure you're splitting on the left most `=` and capturing the whole value, so in case of this label selector:

`label1=value1=value2,label2=value3`

you parse it as:

```
{
  label1: "value1=value2",
  label2: "value3"
}
```

currently this returns:

```
{
  label1: "value1",
  label2: "value3"
}
```

Reply:
Resolved in latest commit.